packages/notification-spa/src/app/app.gql.ts

```diff
@@ -1,4 +1,4 @@
-import gql from 'graphql-tag';
+import { gql } from 'apollo-angular';
```

Comment:
if ```graphql-tag``` is depricated can you please remove from package.json

Reply:
Yes removed graphql form package.json. Thanks @riginoommen